src/main/java/com/gtnewhorizon/gtnhlib/config/ConfigurationManager.java

```diff
@@ -439,4 +447,25 @@ private static void init() {
         configDir = minecraftHome().toPath().resolve("config");
         initialized = true;
     }
+
+    public static void onInit() {
+        if (DUMP_KEYS) {
+            writeLangKeysToFile();
+        }
+    }
+
+    private static void writeLangKeysToFile() {
+        for (Map.Entry<String, List<String>> entry : generatedLangKeys.entrySet()) {
+            val langFile = new File("generated_keys_" + entry.getKey() + ".txt");
```

Comment:
What about dumping keys into files in a subdirectory instead of the root directory? Something like `generated_lang_files/[mod_id].txt`.

Reply:
Sure that works